### PR TITLE
Correctly set type for --proc-path long option

### PR DESCRIPTION
**Pull request by envintus**
* Bugfix: correctly set type for --proc-path long option; should be a… string, not a float
* Bump version number and add CHANGELOG entry

## Pull Request Checklist
#23

#### General

- [X] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [N/A] Update README with any necessary configuration snippets

- [N/A] Binstubs are created if needed

- [x] RuboCop passes

- [N/A] Existing tests pass 

#### New Plugins

- [N/A] Tests

- [N/A] Add the plugin to the README

- [N/A] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

#### Known Compatablity Issues
None

### CODE REVIEW

File: lib/sensu-plugins-cpu-checks/version.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ module SensuPluginsCpuChecks
   module Version
     MAJOR = 1
     MINOR = 0
-    PATCH = 0
+    PATCH = 1
```
Comment (majormoses) [inline lib/sensu-plugins-cpu-checks/version.rb]: can you remove this and I will bump the version after the merge, there are likely several other unreleased fixes.

**Review by majormoses — COMMENTED** [review]
Thanks for the contribution, LGTM after removing the version. After that I can merge and cut a release.

File: lib/sensu-plugins-cpu-checks/version.rb (hunk 1 above)
Comment (envintus) [reply lib/sensu-plugins-cpu-checks/version.rb]: @majormoses Would you also like me to modify my entry in CHANGELOG.md?

Comment (majormoses) [reply lib/sensu-plugins-cpu-checks/version.rb]: yes just remove the version.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## 2017-05-02
```
Comment (majormoses) [inline CHANGELOG.md]: can you remove this line?

**Comment by envintus** [comment]
Done! Thanks for the quick response.

If you're someone to talk to about contributing to these repositories then
I'd like to chat with you should you have time. My email is
hsatterwhite@envintus.com.


[image: FC_MIC2017StandardLogo.png]

[image: TransLoc_logos_wordmark_blue.png]

Hunter Satterwhite

Lead Build & Operations Engineer, TransLoc

252.762.5177 | www.transloc.com

[image: facebook-color-32px.png] <https://www.facebook.com/TransLoc/> [image:
linkedin-color-32px.png] <https://www.linkedin.com/company/transloc> [image:
twitter-color-32px.png] <http://www.twitter.com/transloc> [image:
instagram-color-32px.png] <http://www.instagram.com/transloc_inc>

**Comment by majormoses** [comment]
@envintus thanks after tests pass I can merge this and will try to cut releases tonight on a bunch of sensu plugins, there is a lot of unreleased changes currently. I am trying to spend a bit of time doing some sweeping passes accross the repos but it just takes a bit of time.

**Comment by majormoses** [comment]
@envintus I have reached out to you privately on the process for becoming a maintainer.

**Pull request merged**